lightning/src/chain/keysinterface.rs

```diff
@@ -342,6 +344,18 @@ pub trait KeysInterface: Send + Sync {
 	fn get_channel_id(&self) -> [u8; 32];
 }
 
+#[derive(Clone)]
+/// Holds late-bound channel data
+/// This data is available after the remote accepted the channel.
+pub struct AcceptedChannelData {
```

Comment:
Underscore it can be either counterparty sending us a `accept_channel` or us receiving a  `open_channel` one).

Reply:
reworded

Reply:
Oh oops this probably shouldn't be pub - its never actually used in the public api.